code/game/gamemodes/gang/gang_things.dm

```diff
@@ -18,8 +18,9 @@
 	if(user.mind.has_antag_datum(/datum/antagonist/ert/families))
 		to_chat(user, "As a police officer, you can't join this family. However, you pretend to accept it to keep your cover up.")
 		for(var/threads in team_to_use.free_clothes)
-			new threads(get_turf(user))
-		qdel(src)
+			var/obj/O = new threads(get_turf(user))
+			O.armor = getArmor(7.5, 7.5, 7.5, 7.5, 7.5, 7.5, 7.5, 7.5, 7.5, 7.5)
```

Comment:
Why isn't this in the object's definition?

Reply:
gang uniforms include common clothing items, so we assign it here instead

Reply:
Remove this one as well, please, goof.

Reply:
will do, missed this one